parsers/julia.c

```diff
@@ -0,0 +1,1331 @@
+/*
+*
+*   This source code is released for free distribution under the terms of the
+*   GNU General Public License version 2 or (at your option) any later version.
+*
+*   This module contains functions for generating tags for Julia files.
+*/
```

Comment:
I would like you to add a URL for a document about the syntax of Julia.
I think you refer it during developing this parser.